### ISSUE TITLE
Booking duration

### ISSUE THREAD

**Issue by AGareil** [bgo-bioimagerie/platformmanager#741]
Hi,

It's not a bug, just an ideas.

On PFM, it is possible to regulate the number of reservations per day for a item.
We have resources that are used in days. 
Would it be possible to consider adding a reservation time limitation?
For example, being able to limit the reservation of a portable hard disk to 3 months?

Thanks,

Agnès

**Comment by osallou** [comment]
you mean, for a resource, to specify that booking duration should not exceed X days/... ?

**Comment by AGareil** [comment]
yes, limit the reservation time to 3 months only and if people need it for 5 months, force them to make two different reservations (3months and 2 months). This is to avoid the monopolization of resources which are troubleshooting solution. 
Beyond 3 months, some resources are no longer troubleshooting solutions but working tools and teams must find a more sustainable solution for them.

**Comment by AGareil** [comment]
of course we will allow them to make several reservations of 3 months, but we want this limitation to force them to think about alternative solutions
